hack/build_channel_catalog.sh

```diff
@@ -129,37 +176,28 @@ if [[ ${#skip_versions[@]} -gt 0 ]]; then
     done
 fi
 
-./"$BUNDLE_DEPLOY_DIR"/generate-csv-template.py $generate_csv_template_args > "$csv_template"
-oc process --local -o yaml --raw=true \
-    IMAGE="$OPERATOR_IMAGE" \
-    IMAGE_TAG="$OPERATOR_IMAGE_TAG" \
-    VERSION="$OPERATOR_VERSION" \
-    -f "$csv_template" > "$CSV"
-
-# We need an up-to-date version of opm executable
-opm_local_executable=$(which opm || true)
-if [[ "$opm_local_executable" ]]; then
-    opm_local_version=$(opm version | sed 's/.*OpmVersion:"//;s/".*//')
-fi
-
-if [[ -n "$opm_local_executable" && "$opm_local_version" == "$OPM_VERSION" ]]; then
-    log "Using local opm version $opm_local_executable"
-else
-    opm_download_url="https://github.com/operator-framework/operator-registry/releases/download/$OPM_VERSION/${GOOS}-${GOARCH}-opm"
-    log "Downloading opm from $opm_download_url to $temp_dir/opm"
-    curl -s -L "$opm_download_url" -o "$temp_dir/opm"
-    chmod u+x "$temp_dir/opm"
-    opm_local_executable="$temp_dir/opm"
-fi
+manifests_temp_dir=$(mktemp -d -p "$bundle_temp_dir" manifests.XXXX)
+./hack/generate-operator-bundle-contents.py --name "$OPERATOR_NAME" \
```

Comment:
This assumes the script is invoked from the repo root, right?

To be more explicit, we could compute the directory of this script and use that instead.

```
HERE=${0%/*}
...
$HERE/generate-operator-bundle-contents.py ...
```

Reply:
I assumed that since this is invoked through a `Makefile` target... I can try with the `HERE` approach :)

Reply:
fixed in eb45b68